ledger/sandbox-classic/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is the addition of the new logback configurations intentional?

Reply:
Yes. Without them, the logs for the reset service tests (and other tests) are useless. There's way too much debug logging coming from Hikari, Flyway, etc.

Reply:
I'm fairly sure that this applies to all tests that use the shared `logback.xml`, probably that ought to be fixed instead.

Reply:
Aha, I see. Somehow, Sadbonx doesn't have a _logback.xml_ file. I didn't realize it was only a problem for those and not Sandbox Classic tests.

I suggest we merge this as-is and revisit Sandbox logging in general soon.

Reply:
Took another look and I would definitely prefer to keep this separate, because the production Logback configuration for Sandbox doesn't print timestamps. This is very annoying when debugging, because Bazel corrupts the CI timestamps by holding back logs and then dumping them all at once on failure.

It also logs to a file, which is unnecessary for testing.